examples/quickstart-tensorflow/tfexample/server_app.py

```diff
@@ -2,43 +2,36 @@
 
 from typing import List, Tuple
 
-from flwr.common import Context, Metrics, ndarrays_to_parameters
-from flwr.server import ServerApp, ServerAppComponents, ServerConfig
-from flwr.server.strategy import FedAvg
+from flwr.app import ArrayRecord, ConfigRecord, Context, MetricRecord
+from flwr.serverapp import Grid, ServerApp
+from flwr.serverapp.strategy import FedAvg
 
 from tfexample.task import load_model
 
+# Create the ServerApp
+app = ServerApp()
 
-# Define metric aggregation function
-def weighted_average(metrics: List[Tuple[int, Metrics]]) -> Metrics:
-    # Multiply accuracy of each client by number of examples used
-    accuracies = [num_examples * m["accuracy"] for num_examples, m in metrics]
-    examples = [num_examples for num_examples, _ in metrics]
-
-    # Aggregate and return custom metric (weighted average)
-    return {"accuracy": sum(accuracies) / sum(examples)}
-
-
-def server_fn(context: Context):
-    """Construct components that set the ServerApp behaviour."""
-
-    # Let's define the global model and pass it to the strategy
-    parameters = ndarrays_to_parameters(load_model().get_weights())
-
-    # Define the strategy
-    strategy = strategy = FedAvg(
-        fraction_fit=context.run_config["fraction-fit"],
-        fraction_evaluate=1.0,
-        min_available_clients=2,
-        initial_parameters=parameters,
-        evaluate_metrics_aggregation_fn=weighted_average,
-    )
-    # Read from config
+@app.main()
+def main(grid: Grid, context: Context) -> None:
+    """Main entry point for the ServerApp"""
+    # Load config
     num_rounds = context.run_config["num-server-rounds"]
-    config = ServerConfig(num_rounds=num_rounds)
+    fraction_train = context.run_config["fraction-train"]
+    learning_rate = context.run_config["learning-rate"]
```

Comment:
```suggestion
```
We can remove this line and let the model be initialized with the default lr (in `load_model()`)